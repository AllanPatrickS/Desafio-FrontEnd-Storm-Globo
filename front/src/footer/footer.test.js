import { cleanup } from '@testing-library/react';
import { create } from "react-test-renderer";
import Footer from './footer';

describe('Testing footer', () => {

  it('renders correctly', () => {
    const wrapper = create(<Footer />)
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  afterEach(function () {
    cleanup();
  });

});