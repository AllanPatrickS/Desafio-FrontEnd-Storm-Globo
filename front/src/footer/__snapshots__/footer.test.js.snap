// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing footer renders correctly 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-bar-1 mui-fixed MuiPaper-elevation4"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root makeStyles-orange-2"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
      />
    </svg>
    <p
      className="MuiTypography-root makeStyles-orangeText-3 MuiTypography-body1"
    >
      NO AR
    </p>
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root makeStyles-tv-4"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M21 3H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h5v2h8v-2h5c1.1 0 1.99-.9 1.99-2L23 5c0-1.1-.9-2-2-2zm0 14H3V5h18v12z"
      />
    </svg>
    <p
      className="MuiTypography-root makeStyles-tvText-5 MuiTypography-body1"
    >
      ENCONTRO - 
      
    </p>
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root makeStyles-update-6"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M21 10.12h-6.78l2.74-2.82c-2.73-2.7-7.15-2.8-9.88-.1-2.73 2.71-2.73 7.08 0 9.79s7.15 2.71 9.88 0C18.32 15.65 19 14.08 19 12.1h2c0 1.98-.88 4.55-2.64 6.29-3.51 3.48-9.21 3.48-12.72 0-3.5-3.47-3.53-9.11-.02-12.58s9.14-3.47 12.65 0L21 3v7.12zM12.5 8v4.25l3.5 2.08-.72 1.21L11 13V8h1.5z"
      />
    </svg>
    <p
      className="MuiTypography-root makeStyles-updateText-7 MuiTypography-body1"
    >
      Última atualização em 
      <strong>
        
      </strong>
    </p>
    <p
      className="MuiTypography-root makeStyles-date-8 MuiTypography-body1"
    >
      <strong>
        6
      </strong>
       de 
      <strong>
        Novembro
      </strong>
       de 
      <strong>
        2020
      </strong>
    </p>
    <h4
      className="MuiTypography-root makeStyles-clock-9 MuiTypography-h4"
    >
      <strong>
        ⁄⁄ 
        12
        :
        17
        :
        55
      </strong>
    </h4>
  </div>
</header>
`;
